<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapEditorClass</class>
 <widget class="QMainWindow" name="MapEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MapEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="ViewSettingsWidget" native="true">
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="gb_ViewMode">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>Просмотр</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="radioButton_5">
            <property name="text">
             <string>Общий вид</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton">
            <property name="text">
             <string>Рельеф</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_2">
            <property name="text">
             <string>Вода</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gb_Scale">
         <property name="title">
          <string>Масштаб</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QSpinBox" name="spinBox_Scale">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="value">
             <number>4</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_Scale">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="pageStep">
             <number>5</number>
            </property>
            <property name="value">
             <number>5</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_ResetScale">
            <property name="text">
             <string>Сброc</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gb_ViewSettings">
         <property name="title">
          <string>Отображение</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="cb_PaintGrid">
            <property name="text">
             <string>Сетка</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_PaintInfo">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Значения</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_Directions">
            <property name="text">
             <string>Направления</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gb_MapsList">
         <property name="title">
          <string>Список карт</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="MapQListView" name="MapListView">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>120</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>88</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="MapEditorGraphicsView" name="mainView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>350</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="EditSettingsWidget" native="true">
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>Редактирование</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QRadioButton" name="radioButton_6">
            <property name="text">
             <string>Поверхность</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_7">
            <property name="text">
             <string>Вода</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>Вода</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QCheckBox" name="checkBox_4">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Подземная</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gb_MouseBrashSettings">
         <property name="title">
          <string>Кисть</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Размер</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinBox_BrushSize">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Сила</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBox_BrushStrength">
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QGroupBox" name="gb_MouseBrushDirection">
            <property name="title">
             <string>Значение</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QRadioButton" name="rb_MouseBrashInc">
               <property name="text">
                <string>Увеличить</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rb_MouseBrashDec">
               <property name="text">
                <string>Уменьшить</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>106</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>766</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Карта</string>
    </property>
    <addaction name="action_CreateNewMap"/>
    <addaction name="action_LoadMap"/>
    <addaction name="action_SaveMap"/>
    <addaction name="action_CloseMap"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="action_ViewSettingsVisible"/>
    <addaction name="action_MapsListVisible"/>
    <addaction name="action_EditSettingsVisible"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="MapStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>Новая</string>
   </property>
  </action>
  <action name="action_LoadMap">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Загрузить</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_CreateNewMap">
   <property name="text">
    <string>Новая</string>
   </property>
  </action>
  <action name="action_SaveMap">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action_EditSettingsVisible">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Редактор</string>
   </property>
  </action>
  <action name="action_ViewSettingsVisible">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Настройки просмотра</string>
   </property>
  </action>
  <action name="action_CloseMap">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </action>
  <action name="action_MapsListVisible">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Список карт</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MapStatusBar</class>
   <extends>QStatusBar</extends>
   <header location="global">mapstatusbar.h</header>
  </customwidget>
  <customwidget>
   <class>MapEditorGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>mapeditorgraphicsview.h</header>
  </customwidget>
  <customwidget>
   <class>MapQListView</class>
   <extends>QListView</extends>
   <header location="global">mapqlistview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>action_ViewSettingsVisible</sender>
   <signal>toggled(bool)</signal>
   <receiver>ViewSettingsWidget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_EditSettingsVisible</sender>
   <signal>toggled(bool)</signal>
   <receiver>EditSettingsWidget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>612</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_Scale</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_Scale</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>72</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_Scale</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_Scale</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>72</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>MapEditorClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_MapsListVisible</sender>
   <signal>toggled(bool)</signal>
   <receiver>gb_MapsList</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
